samples/files.js

```diff
@@ -312,6 +313,70 @@ async function generateSignedUrl(bucketName, filename) {
   // [END storage_generate_signed_url]
 }
 
+async function generateV4ReadSignedUrl(bucketName, filename) {
+  // [START storage_generate_signed_url_v4]
+  // Imports the Google Cloud client library
+  const {Storage} = require('@google-cloud/storage');
+
+  // Creates a client
+  const storage = new Storage();
+
+  /**
+   * TODO(developer): Uncomment the following lines before running the sample.
+   */
+  // const bucketName = 'Name of a bucket, e.g. my-bucket';
+  // const filename = 'File to access, e.g. file.txt';
+
+  // These options will allow temporary read access to the file
+  const options = {
+    version: 'v4',
+    action: 'read',
+    expires: Date.now() + 1000 * 60 * 60, // one hour
+  };
+
+  // Get a v4 signed URL for reading the file
+  const [url] = await storage
+    .bucket(bucketName)
+    .file(filename)
+    .getSignedUrl(options);
+
+  console.log(`The v4 signed url for reading ${filename} is ${url}`);
+  // [END storage_generate_signed_url_v4]
+}
+
+async function generateV4UploadSignedUrl(bucketName, filename) {
+  // [START storage_generate_upload_signed_url_v4]
+  // Imports the Google Cloud client library
+  const {Storage} = require('@google-cloud/storage');
+
+  // Creates a client
+  const storage = new Storage();
+
+  /**
+   * TODO(developer): Uncomment the following lines before running the sample.
+   */
+  // const bucketName = 'Name of a bucket, e.g. my-bucket';
+  // const filename = 'File to access, e.g. file.txt';
+
+  // These options will allow temporary uploading of the file with outgoing
+  // Content-Type: application/octet-stream header.
+  const options = {
+    version: 'v4',
+    action: 'write',
+    expires: Date.now() + 1000 * 60 * 60, // one hour
+    contentType: 'application/octet-stream',
+  };
+
+  // Get a v4 signed URL for uploading file
+  const [url] = await storage
+    .bucket(bucketName)
+    .file(filename)
+    .getSignedUrl(options);
+
+  console.log(`The v4 signed url for uploading ${filename} is ${url}`);
```

Comment:
Could you update output to follow Java's output?

https://github.com/googleapis/google-cloud-java/blob/v4support/google-cloud-examples/src/main/java/com/google/cloud/examples/storage/snippets/StorageSnippets.java#L1536-L1542